### ISSUE TITLE
SDR LS factor is  incorrect somehow

### ISSUE THREAD

**Issue by richpsharp** [natcap/invest#530]
Best way to see is to compare it against GRASS

**Comment by dcdenu4** [comment]
Hey @richpsharp , do you have a good feel for how this effects the SDR outputs? There's a few Forum threads on SDR going and was curious if this would manifest itself noticeably in the outputs.

**Comment by richpsharp** [comment]
yes it would! if it's easy can you link that thread here? it's not always easy for me to find what thread people are talking about in the forms.

**Comment by richpsharp** [comment]
There's no bug after all!

**Comment by emlys** [comment]
Hey @richpsharp I've just been communicating with a user about [this post](https://community.naturalcapitalproject.org/t/usle-output-generated-from-invest-sdr/1820) and I suspect something's up with the `m` factor.
Here's the function for `m` when slope > 9 degrees (0.089 radians): https://www.wolframalpha.com/input/?i=%28%28sin%28x%29%2F0.0986%29%2F%283*sin%28x%29%5E0.8%2B0.56%29%29%2F%281+-+%28sin%28x%29%2F0.0986%29%2F%283*sin%28x%29%5E0.8%2B0.56%29%29%2C+x+from+0.089+to+0.785
It doesn't seem right that `m` goes 0.2 for slope <= 1%; 0.3 for 1% - 3.5%; 0.4 for 3.5% - 5%; 0.5 for 5% - 9%; and then it's negative for >9%.
Is this the same thing you were looking at?

**Comment by richpsharp** [comment]
Hi @emlys, that's scary enough that I was trying to figure out what's going on. I certainly see wolfram doing that negative plot, but everything that I check myself suggests that the plot is wrong, including calculating numbers by hand. Here's what I've got:

A spreadsheet that does step by step analysis based on the equation I copied out of your link just in case there were typos (there aren't): https://docs.google.com/spreadsheets/d/1K8PexITJsQ4HLlfOOxa9sZ4evcgpCkEeuTHK_ZBdj9g/edit?usp=sharing

And a python script showing the same results in a matplotlib: https://gist.github.com/richpsharp/fe1bb9030168a925aed2b8b57a4b5975

So I'm not sure what's going on with wolfram but I think beta and m are correct in the scientific and software implementations.
